.agents/skills/e2e-container/scripts/install-prerequisites.sh

```
@@ -0,0 +1,46 @@
+#!/bin/bash
+# Install prerequisites for running E2E tests in a Linux container.
+
+set -euo pipefail
+
+# Go PATH
+if ! echo "$PATH" | grep -q "$(go env GOPATH)/bin"; then
+  export PATH="$PATH:$(go env GOPATH)/bin"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)